### DISCUSSION TITLE
How to use Master branch files

### DISCUSSION THREAD

**Discussion by yann-mrt** [Courseplay/courseplay#7249] [Q&A] [answered]
Good day,
I would like to know how to use the files that are in the master branch.
I understand that is may contain bug or else.
Thanks
Yannick

**Comment by Tensuko** [comment, marked as answer]
green button -> download .zip
rezip the files inside the folder of the .zip file.

I rather suggest you to stick to the modhub release!

**Reply by yann-mrt** [reply]
Thank you, I will !
